config/routes.rb

```diff
@@ -1,6 +1,6 @@
 Rails.application.routes.draw do
   root to: "wishlists#index"
-  resources :wishlists, only: [:index, :new, :create, :destroy]
+  resources :wishlists, only: %i(index new create destroy edit update)
```

Comment:
Yeah, I'm not complaining. I knew it was inevitable. Just a reference to our previous conversation about this precise line. 😜